CMakeLists.txt

```diff
@@ -1625,6 +1625,8 @@ hpx_check_for_mm_prefetch(DEFINITIONS HPX_HAVE_MM_PREFETCH)
 
 hpx_check_for_stable_inplace_merge(DEFINITIONS HPX_HAVE_STABLE_INPLACE_MERGE)
 
+hpx_check_for_pthread_setname_np(DEFINITIONS HPX_HAVE_PTHREAD_SETNAME_NP)
+
```

Comment:
Could you please move this up or down? Currently it's placed in between the feature test for `HPX_HAVE_STABLE_INPLACE_MERGE` and it's use below.